src/tmpfiles/tmpfiles.c

```diff
@@ -2718,6 +2886,13 @@ static int parse_line(
         if (r < 0)
                 return r;
 
+        /*
+         * The "age-by" argument considers all
+         * file timestamp types by default.
+         */
+        i.age_by_file = AGE_BY_DEFAULT_FILE;
+        i.age_by_dir = AGE_BY_DEFAULT_DIR;
```

Comment:
i think static initializations like this should best be done when the variable is declared, i.e. the beginning of this function, change:

```c
_cleanup_(item_free_contents) Item i = {};
```

to

```c
_cleanup_(item_free_contents) Item i = {
        .age_by_file = AGE_BY_DEFAULT_FILE,
        .age_by_dir = AGE_BY_DEFAULT_DIR,
};
```